### PR TITLE
Minimum viable product jc

### PR DESCRIPTION
**Pull request by jclif**

### CODE REVIEW

File: MVP.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+# Minimum Viable Product
+
+## (Provisional) Top-level goals
+- Examine player on champion-by-champion basis.
+- Compare victory-relavent statistics with tier above the current one.
```
Comment (aegatlin) [inline MVP.md]: Typo. "relevant"

File: MVP.md [hunk 2]
```diff
@@ -0,0 +1,8 @@
+# Minimum Viable Product
+
+## (Provisional) Top-level goals
+- Examine player on champion-by-champion basis.
+- Compare victory-relevent statistics with tier above the current one.
```
Comment (aegatlin) [inline MVP.md]: lol... "relevant"

Comment (jclif) [inline MVP.md]: :fu:

**Comment by jclif** [comment]
fixes #4

**Pull request merged**